### ISSUE TITLE
That Library have java version?

### ISSUE THREAD

**Issue by cemalsayin** [hoanganhtuan95ptit/DraggablePanel#3]
Hello, i want to create a app like that example but my project created with java. I found ["pedrovgs/DraggablePanel"](https://github.com/pedrovgs/DraggablePanel) but it archived. I cant use it. Can i found that library's java version?

**Comment by zenosama1111** [comment]
> Hello, i want to create a app like that example but my project created with java. I found ["pedrovgs/DraggablePanel"](https://github.com/pedrovgs/DraggablePanel) but it archived. I cant use it. Can i found that library's java version?

My project was created with java too. I think this library is both kotlin and java supported.

**Comment by cemalsayin** [comment]
> > Hello, i want to create a app like that example but my project created with java. I found ["pedrovgs/DraggablePanel"](https://github.com/pedrovgs/DraggablePanel) but it archived. I cant use it. Can i found that library's java version?
> 
> My project was written with java too. I think this library is both kotlin and java supported.

yes it supported java but it was archived. i'd compile library i applied all steps but it gives some errors. A lot of people ask some question about which errors but it isn't any answer.

**Comment by hoanganhtuan95ptit** [comment]
> > > Hello, i want to create a app like that example but my project created with java. I found ["pedrovgs/DraggablePanel"](https://github.com/pedrovgs/DraggablePanel) but it archived. I cant use it. Can i found that library's java version?
> > 
> > 
> > My project was written with java too. I think this library is both kotlin and java supported.
> 
> yes it supported java but it was archived. i'd compile library i applied all steps but it gives some errors. A lot of people ask some question about which errors but it isn't any answer.

Thanks for your interest library, my library supports both Java and Kotlin. Can you send me the error information you encountered to find the resolution direction, thank you

**Comment by zenosama1111** [comment]
> > > > Hello, i want to create a app like that example but my project created with java. I found ["pedrovgs/DraggablePanel"](https://github.com/pedrovgs/DraggablePanel) but it archived. I cant use it. Can i found that library's java version?
> > > 
> > > 
> > > My project was written with java too. I think this library is both kotlin and java supported.
> > 
> > 
> > yes it supported java but it was archived. i'd compile library i applied all steps but it gives some errors. A lot of people ask some question about which errors but it isn't any answer.
> 
> Thanks for your interest library, my library supports both Java and Kotlin. Can you send me the error information you encountered to find the resolution direction, thank you

I tried this library and I don't see any errors in java. I am so glad that there was another library for DraggablePanel starting this year. In my case, the second frame fragment of the panel is merging to main fragment of my MainActivity. I tried to convert the kotlin code from the example to my java project but still no luck.   I'm struggling to implement this library in java since there is no java tutorial in it.

**Comment by devepoperjitendra07** [comment]
Hi please add closeToMaxAnimation
